docs/spark/tutorials/hdinsight-deployment.md

```diff
@@ -9,124 +9,180 @@ ms.custom: mvc
+> [!IMPORTANT]  
+> Billing for HDInsight clusters is prorated per minute, whether you are using them or not. Be sure to delete your cluster after you have finished using it. For more information, see the [Clean up resources](#clean-up-resources) section of this article.
 
-2. Publish your Spark .NET app as self-contained.
+1. Visit the [Azure portal](https://portal.azure.com).
 
-   You can run the following command on Linux.
+2. Select **+ Create a resource**. Then select **HDInsight** from **Analytics**.
 
-   ```dotnetcli
-   dotnet publish -c Release -f netcoreapp2.1 -r ubuntu.16.04-x64
+    ![Create HDInsight resource from Azure portal](./media/hdinsight-deployment/create-hdinsight-resource.png)
+
+3. Under **Basics**, provide the following values:
+
+    |Property  |Description  |
+    |---------|---------|
+    |Subscription  | From the drop-down, choose one of your active Azure subscriptions. |
+    |Resource group | Specify whether you want to create a new resource group or use an existing one. A resource group is a container that holds related resources for an Azure solution. |
+    |Cluster name | Give a name to your HDInsight Spark cluster.|
+    |Location   | Select a location for the resource group. The template uses this location for creating the cluster as well as for the default cluster storage. |
+    |Cluster type| Select **Spark** as the cluster type.|
+    |Cluster version|This field will autopopulate with the default version once the cluster type has been selected.|
+    |Cluster login username| Enter the cluster login username.  The default name is *admin*. |
+    |Cluster login password| Enter any login password. |
+    |Secure Shell (SSH) username| Enter the SSH username. By default, this account shares the same password as the *Cluster Login username* account. |
+
+4. Select **Next: Storage >>** to continue to the **Storage** page. Under **Storage**, provide the following values:
+
+    |Property  |Description  |
+    |---------|---------|
+    |Primary storage type|Use the default value **Azure Storage**.|
+    |Selection method|Use the default value **Select from list**.|
+    |Primary storage account|Choose your subscription and one of your active storage accounts within that subscription.|
+    |Container|This container is the specific blob container in your storage account where your cluster looks for files to run your app in the cloud. You can give it any available name.|
+
+5. Under **Review + create**, select **Create**. It takes about 20 minutes to create the cluster. The cluster must be created before you can continue to the next session.
+
+## Publish your app
+
+Next, you publish the *mySparkApp* created in the [.NET for Apache Spark - Get Started in 10-Minutes tutorial to ensure your Spark cluster has access to all the files it needs to run your app. 
+
+1. Run the following commands to publish the *mySparkApp*.
+
+   **On Windows:**
+
+   ```cmd
```

Comment:
@mairaw, same in this PR too, should it be `dotnetcli` or `console` ?

Also the other occurrences of `cmd` should be changed to `console` ?

Reply:
Not sure. The dotnetcli will only color .NET Core CLI commands. This one involves other commands such as cd.
But yes, we’ve been using console instead of cmd.